### ISSUE TITLE
Need Fauna 101 sample

### ISSUE THREAD

**Issue by csharpfritz** [fauna/faunadb-csharp#122]
I'm new to Fauna and having a VERY hard time understanding how to query and return a collection of objects.

In EntityFramework I would access a property on my database connection like:

```
dbContext.MyCollectionName
```

and be able to interact with the contents with LINQ syntax.  I struggle to find the right combination of Query(Paginate(Match()))).To<MyObject>().Value to get objects read from Fauna.  Can we put together a simple example document that shows Fauna 101 for each of the CRUD operations?

**Comment by evbruno** [comment]
Hi there, welcome to Fauna :)

Did you take a look at our tutorials ? 

https://docs.fauna.com/fauna/current/tutorials/crud?lang=csharp

**Comment by csharpfritz** [comment]
Yes, and I struggled with them for 3 hours.  They are not linked on the homepage of this repository, it was very hard to find and does not cover the scenario I'm looking for help with.

**Comment by RamblingGeekUK** [comment]
I thought it was just me

**Comment by evbruno** [comment]
Isn't [that example ](https://github.com/fauna/faunadb-csharp#how-to-work-with-user-defined-classes) working for you guys? How do you think we could improve it?

**Comment by RamblingGeekUK** [comment]
For me the example the data comes back as and there doesn't appear to be away to make it an object ?   I thought this was going to be the way https://docs.fauna.com/fauna/current/api/fql/functions/toobject?lang=csharp

**Comment by evbruno** [comment]
btw, I would like to invite you guys to join our Forums at https://forums.fauna.com/

**Comment by codewisdom** [comment]
No offense, but this API is far too obtuse to be exciting for any developer trying to explore a new technology.  One of the first things anyone tests is retrieval of a list, largely because many of us know that is not straight forward in something like DynamoDB.  This was of no help in getting to that point:

https://docs.fauna.com/fauna/current/api/fql/functions/toobject?lang=csharp

I finally figured it out, by referencing this example: 

https://github.com/fldsblzs/fauna-aspnet-sample

I examined the payload returned by the this client API.  Who thought adding all that cruft where you have to navigate layers to get the ref Id for each item on a list was a good idea?  That's unnecessary string parsing and it most definitely would impact performance if retrieving a large list, which may be appropriate in some cases.  Is there a way to query fauna to trim that down?  If so, please let me know.
